deployments/root/conda-envs/alchemiscale-compute.yml

```diff
@@ -5,16 +5,22 @@ channels:
 dependencies:
   - pip
   - python =3.12
-  - cudatoolkit =11.8
+  - cuda-version >=12
```

Comment:
Had to hard-force an update on my workstation for this to work (to drivers 550). I would suspect most folks are lagging by quite a lot on driver updates, so pinning an upper version or even avoiding v12 might be wise for an initial release (I understand the need for reduced deployment size though).

Reply:
This actually needs to match the requirements of the HPC resources we run the compute workers on, this isn't the general purpose "public" version, that would be here: https://github.com/OpenFreeEnergy/alchemiscale/blob/main/devtools/conda-envs/alchemiscale-compute.yml#L8

Reply:
Indeed, I'd like to try out `cuda` 12 and identify what the showstoppers are, if any, before we fall back to 11.8.

Reply:
Sorry here by "most folks" I mean "most HPC centers" - I'm doing a local run to check that the compute environment does actually.. compute.

Reply:
I can verify that 12.4 runs fine on my workstation, so at least "will cuda 12.x work" seems good.